pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -933,71 +918,6 @@ spec:
 			name:  "sa-step",
 			cmd:   "/mycmd",
 		}}),
-	}, {
-		name:    "params",
-		taskRun: taskRunSubstitution,
```

Comment:
`taskRunSubstitution` hasn't been deleted; I think this test case should remain

Reply:
I think for this case, the expected steps and volumes will be (almost 🤔  )completely changed without imageResourceType as the outputs resources. Would it make sense to delete the `taskRunSubstitution` also since they are only referenced here?

Reply:
It looks like it gets some volumes from the image resource and some from the git resource. Can you just remove the ones that it gets from the image resource? (and same for params)

Reply:
That makes sense for maintaining code coverage:) Thanks!